mite_cli.py

```diff
@@ -240,20 +252,21 @@ def edit(mite, id, date, minutes, project_id, service_id, note):
 
     print("Entry edited for {}!".format(date))
 
+
 # from-date and to-date are strings, so I think its okay to support
 # special keys such as yesterday and today.
 @cli.command()
-@click.option("--from-date", default="yesterday", help="From date"
-              " (default is yesterday)")
-@click.option("--to-date", default="today", help="To date"
-              " (default is today)")
+@click.option(
+    "--from-date", default="yesterday", help="From date" " (default is yesterday)"
+)
+@click.option("--to-date", default="today", help="To date" " (default is today)")
```

Comment:
Don't need these string splits anymore.

Reply:
Never mind. I am going to edit that help out and use `show_default` option from click.